src/utils/utils.ts

```diff
@@ -179,7 +179,7 @@ export function getProperty (value: { [key: string]: any}, key: string): any {
   }
   
   const keys: string[] = key.split('.');
-  let result: any = value[keys.shift()];
+  let result: any = value[keys.shift()!];
```

Comment:
Would you mind publishing 6.4.0 on npm? AFAICS only 6.3.0 is published currently.